src/__tests__/components/Level.test.js

```diff
@@ -2,30 +2,18 @@ import React from 'react';
 import {render} from '@testing-library/react';
 import Level from '../../components/level';
 
-const states = [
-  {
-    deltaconfirmed: '10',
-    deltadeaths: '3',
-    deltarecovered: '5',
-  },
-  {
-    active: '75',
-    confirmed: '83',
-    deaths: '3',
-    recovered: '5',
-    state: 'Karnataka',
-  },
-  {
-    active: '5',
-    confirmed: '3',
-    deaths: '1',
-    recovered: '2',
-    state: 'Gujarat',
-  },
-];
+const data = {
```

Comment:
Can you change the name of the variable to `state`? As we are passing `states[0]` in the code. 
It explains it better while reading the tests.

Reply:
Changed them :+1: